modules/minimum-required-permissions-ipi-azure.adoc

```diff
@@ -146,6 +146,17 @@ The following permissions are not required to create the private {product-title}
 * `Microsoft.Storage/storageAccounts/write`
 ====
 
+.Required permissions for creating a private image registry endpoint
```

Comment:
since the step to configure internal image-registry is optional, here should be "Optional permission" instead of "Required permission", so that user only requires those permissions when configuring the step.

suggest to change `creating a private image registry endpoint` to `creating a image registry storage account private endpoint`, @flavianmissi  wdyt?

Reply:
That does make sense @jinyunma !
It's worth mentioning that I added these permissions to the registry operator credentials request: https://github.com/openshift/cluster-image-registry-operator/pull/971/files
If we are to merge this then I think we won't need to document this.
WDYT?